Fixes for minor issues in mobile docs (#707)
aerogear/mobile-docs: modules/ROOT/pages/getting-started-configuring.adoc

:service-name: Identity Management

=== Configuring {idm-name}

. Log into the {mobile-developer-console} and navigate to the {mobile-client} screen.

. Select the *Mobile Services* tab and if a spinning icon is displayed to the right of the {keycloak-service} entry, wait for the binding process to complete.

. If the _{idm-name} Realm URL_ URL is not visible, expand the Identity Management Service by clicking the > icon.

. Click on the *{idm-name} Realm URL* link to open the {idm-name} Administration Console.
+
// TODO: @wei will default credentials always be as follows?
. Log in to the Administration console using the credentials you specified when xref:provisioning[Provisioning] the service (defaults to admin/admin).

. Select *Clients* from the left navigation menu.
+
// TODO: with a cordova app, I didn't see mobile development platform
. Select your client from the list of clients. The name of your client is derived from the name of the {mobile-client}, the name of the mobile development platform and the client type, for example `myapp-android-public`.

> . Add an additional entry to *Valid Redirect URIs*.

Reviewer: ```suggestion
. Add an additional entry to the *Valid Redirect URIs* input field.
```

:service-name: Identity Management

=== Configuring {idm-name}

. Log into the {mobile-developer-console} and navigate to the {mobile-client} screen.

. Select the *Mobile Services* tab.

> . If a binding to the {keycloak-service} service is in progress, a spinning icon is displayed to the right of the {keycloak-service} entry. Wait for the binding process to complete.

Reviewer: Combine this step and the step above into one step:

". Select the *Mobile Services* tab and if a spinning icon is displayed to the right of the {keycloak-service} entry, wait for the binding process to complete."

Author: Should be fixed now.

. If the _{idm-name} Realm URL_ URL is not visible, expand the Identity Management Service by clicking the > icon.

. Click on the *{idm-name} Realm URL* link to open the {idm-name} Administration Console.
+
// TODO: @wei will default credentials always be as follows?
. Log in to the Administration console using the credentials you specified when xref:provisioning[Provisioning] the service (defaults to admin/admin).

> . Select `Clients` from the left navigation menu.

Reviewer: ```suggestion
. Select *Clients* from the left navigation menu.
```

+
// TODO: with a cordova app, I didn't see mobile development platform
. Select your client from the list of clients. The name of your client is derived from the name of the {mobile-client}, the name of the mobile development platform and the client type, for example `myapp-android-public`.

> . Add an additional entry to *Valid Redirect URIs*.

Reviewer: Is it an "input text" filed or what is it that you add the additional entry too?

Author: The `Valid Redirect URIs` supports multiple "input text" fields to be added to it.

Author: ![Snímek obrazovky 2019-06-17 v 10 18 40](https://user-images.githubusercontent.com/13137961/59589075-66eb7f00-90e9-11e9-87c4-8b3edfcd0109.png)

Reviewer: Proposal made here https://github.com/aerogear/mobile-docs/pull/707#discussion_r294193666

Cordova::
+
--
// end::excludeDownstream[]
Set the additional value to `http://localhost*`. 
// tag::excludeDownstream[]
--
Xamarin::
+
--
Depending on the platform, set the redirect URL as described in either the Android or the iOS tab.
--
// end::excludeDownstream[]
====

. Add an additional entry to *Web Origins*.
+
[tabs]
====
// tag::excludeDownstream[]
Android::
+
--

> Use the package name of the Android app to avoid conflicts. (e.g. `com.aerogear.androidshowcase`)

Reviewer: ```suggestion
Use the package name of the Android app to avoid conflicts, for example, `com.aerogear.androidshowcase`.
```

--
iOS::
+
--

> Use the Bundle Identifier of the iOS app. (e.g. `org.aerogear.ios-showcase-template`)

Reviewer: ```suggestion
Use the Bundle Identifier of the iOS app, for example, `org.aerogear.ios-showcase-template`.
```

--
Cordova::
+
--
// end::excludeDownstream[]

> Set additional value to `http://localhost*`. 

Reviewer: ```suggestion
Set the additional value to `http://localhost*`. 
```

// tag::excludeDownstream[]
--
Xamarin::
+
--
Depending on the target platform, set the additional value as described in either the Android or the iOS tab.
--
// end::excludeDownstream[]
====

. Save your changes.

> . Creating a new user account as described in link:{docs-create-user}[Creating a New User].

Reviewer: ```suggestion
. Create a new user account as described in link:{docs-create-user}[Creating a New User].
```

> . Set up credentials for the new user as described in link:{docs-user-creds}[User Credentials].

Reviewer: ```suggestion
. Create credentials for the new user as described in link:{docs-user-creds}[User Credentials].
```
